scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/AvailableConnections.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *    Achim Kraus (Bosch Software Innovations GmbH) - initial implementation
+ ******************************************************************************/
+package org.eclipse.californium.scandium.dtls;
+
+/**
+ * Pair of connections.
+ * 
+ * Helper to prevent duplicated calls to
+ * {@link ResumptionSupportingConnectionStore#find(SessionId)}.
+ * 
+ * <ul>
+ * <li>connection associated with the peer's address or cid</li>
+ * <li>connection associated with the session id</li>
+ * </ul>
+ */
+public final class AvailableConnections {
```

Comment:
Even if I can not see why somebody would like to inherit from this class, I prefer to  avoid final keyword for class.

Reply:
OK